server/ws.py

```diff
@@ -3,16 +3,17 @@
 from persona_link.avatar import call_webhook
 from persona_link.avatar.models import Avatar, AvatarInput, WebhookResponseData
 
-from .app import construct_and_send, get_message_count
+
 from .models import Conversation
 
 router = APIRouter()
 
 connections: dict[str, WebSocket] = {}
 
 
-@router.websocket("/{conversation_id}")
+@router.websocket("/{conversation_id}/")
 async def websocket_endpoint(websocket: WebSocket, conversation_id: str):
+    from .app import construct_and_send, get_message_count
```

Comment:
Instead create some. utils.py and put these methods there. Then both app.py and models.py should import from there.